app/helpers/sessions_helper.rb

```diff
@@ -16,7 +16,7 @@ def current_user
     elsif user_id = cookies.signed[:user_id]
       user = User.find_by id: user_id
 
-      if user && user.authenticated?(:remember, cookies[:remember_token])
+      if user&.authenticated?(:remember, cookies[:remember_token])
```

Comment:
&. là gì vậy

Reply:
user && user.authenticated?  =  user&.authenticated?
chi viet khac di thoi anh a.

Reply:
sai

Reply:
&.  là cách viết an toàn thông thường để đảm bảo không có các exception không mong muốn bắn ra khi object bị nil.